modules/ztp-telco-ran-software-versions.adoc

```diff
@@ -1,31 +1,57 @@
 // Module included in the following assemblies:
 //
 // * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc
+// * telco_ref_design_specs/ran/telco-ran-ref-validation-artifacts.adoc
 
-:_content-type: CONCEPT
+:_content-type: REFERENCE
 [id="ztp-telco-ran-software-versions_{context}"]
-= Telco RAN {product-version} validated solution software versions
+= Telco RAN {product-version} validated software components
 
-The Red Hat Telco Radio Access Network (RAN) version {product-version} solution has been validated using the following Red Hat software products.
+The Red Hat Telco RAN {product-version} solution has been validated using the following Red Hat software products for {product-title}.
 
-.Telco RAN {product-version} validated solution software
+.Telco RAN DU {product-version} validated software components
 [cols=2*, width="80%", options="header"]
 |====
-|Product
+|Component
 |Software version
 
-|Hub cluster {product-title} version
-|4.13
+|RAN DU cluster version
+|4.14
```

Comment:
Is there a reason these are 4.14 and not {product-version} just curious

Reply:
The RAN software stack isn't necessarily tied to 4.14, though it generally tracks OCP Core.